tests/rule_based_profiler/bobby_user_workflow_fixture.py

```diff
@@ -342,14 +349,14 @@ def bobby_columnar_table_multi_batch():
                         }
                     }
                 },
-            },
+            }
         ),
         ExpectationConfiguration(
             **{
                 "kwargs": {
                     "column": "fare_amount",
-                    "min_value": -76.43,
-                    "max_value": 3.43,
+                    "min_value": -92.96,
+                    "max_value": 0.0,
```

Comment:
This seems intuitively broken...

Reply:
Did we give up on sign detection?

Reply:
@jcampbell This is caused by the fact that we have negative fairs and surcharges in our data set.  This has been confirmed throughout the 36 months of Taxi trip data that we have.  What we can do here is build another data set, where such meaningless data is not allowed so as to enable us to get a better qualitative interpretation of our tests.  Please let me know -- how to discuss the best way to proceed.  Thanks.

Reply:
Oh real data, how I love thee.